specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2025-08-01-preview/examples/RedisEnterpriseMigrationStart.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "clusterName": "cache1",
+    "resourceGroupName": "rg1",
+    "api-version": "2025-08-01-preview",
+    "subscriptionId": "e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f",
+    "parameters": {
+      "sourceResourceId": "/subscriptions/e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f/resourceGroups/rg1/providers/Microsoft.Cache/redis/cache1",
+      "switchDns": true,
+      "skipDataMigration": true
+    }
```

Comment:
if migrations is a resource that we create with PUT, then this payload needs to follow the resource conventions and have any custom properties inside the 'properties' property.